contribute/contribute.rst

```diff
@@ -418,10 +421,19 @@ Consider having multiple terminals open simultaneously.
 - One for running commands on the VM.
 - Another for browsing through the logs.
 
-When making backend changes, you may want to view/tail logs.
-Everything that your code or any rockstor service logs goes into the following logs:
+When making backend changes, you may want to view/:code:`tail` logs.
+By default, Rockstor's log level is set to INFO.
+To increase the log level to DEBUG, you can use the following script:
 
-.. code-block:: console
+.. code-block:: text
+
+        buildvm:~ # cd /opt/rockstor
+        buildvm:~ # export DJANGO_SETTINGS_MODULE=settings
+        buildvm:~ # poetry run debug-mode ON
+
+Everything that your code or any Rockstor service logs goes into the following files:
```

Comment:
Thanks for this. I'm surprised we didn't already have this!

Reply:
Our buildout method actually automatically set `DEBUG` to `True` in settings.py. I forgot exactly how, but I remember seeing it. What do you think of adding an optional flag to `build.sh` to do that (leaving defaults as is, of course)?

Reply:
> What do you think of adding an optional flag to `build.sh` to do that (leaving defaults as is, of course)?

That sound like the way to go, given we use the same script when building the package. Plus it will 'surface' this debug element for those that read no further than the build.sh itself. Which brings to mind the idea that we include a referenced to our developer docs within build.sh itself. So folks can easily find this doc section with only a casual look at build.sh.

Reply:
I do like the idea of adding a link to the docs in build.sh itself!